.github/workflows/ci.yml

```diff
@@ -37,6 +37,6 @@ jobs:
         with:
           go-version: "1.21"
       - name: golangci-lint
-        uses: golangci/golangci-lint-action@3cfe3a4abbb849e10058ce4af15d205b6da42804 # v3.7.1
+        uses: golangci/golangci-lint-action@3cfe3a4abbb849e10058ce4af15d205b6da42804 # v4.0.0
```

Comment:
both 3.7.1 and 4.0.0 point to the same commit 3cfe3a4abbb849e10058ce4af15d205b6da42804